.github/workflows/refresh_seeded_grants.yml

```diff
@@ -0,0 +1,46 @@
+name: Refresh seeded grants
+on:
+  # the schedule is optional, but helps to make sure no dangling resources are left when GitHub Action does not behave properly
+  schedule:
+    - cron: "30 2 * * *"
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    if: github.event_name == 'schedule'
```

Comment:
will this stop it from running on manual workflow dispatch? sorry I should just look into a bit more but hopefully an easy thing to short circuit

Reply:
😭